### PR TITLE
Merge gpos into gporca

### PR DESCRIPTION
**Pull request by hsyuan**
GPOS is not a stand alone library anymore. All the source code of gpos is merged into gporca. So some changes are required in gpdb.

### CODE REVIEW

**Comment by hlinnaka** [comment]
Sounds great!

**Comment by danielgustafsson** [comment]
👍  🎉

File: src/backend/gpopt/utils/funcs.cpp [hunk 1]
```diff
@@ -125,7 +123,7 @@ StringInfo
 OptVersion()
 {
 	StringInfo str = gpdb::SiMakeStringInfo();
-	appendStringInfo(str, "%d.%d", GPORCA_VERSION_MAJOR, GPORCA_VERSION_MINOR);
+	appendStringInfo(str, "%s", GPORCA_VERSION_STRING);
```
Comment (d) [inline src/backend/gpopt/utils/funcs.cpp]: This is AWESOME.

**Comment by d** [comment]
This is awesome. Let's wait until [ORCA CI](https://ci.orca.pivotalci.info/teams/main/pipelines/gporca/jobs/orca_publish_tag/builds/113) publishes the release and merge this. (I guess that's after lunch)

**Pull request merged**